### ISSUE TITLE
Add Typesense support

### ISSUE THREAD

**Issue by timkelty** [riasvdv/craft-scout#183]
https://typesense.org/

**Comment by zizther** [comment]
@timkelty Is this something you are still considering, as this would be quite a good option alongside Algolia

**Comment by timkelty** [comment]
Absolutely! Haven't had the time, but something to keep in mind, @janhenckens

**Comment by janhenckens** [comment]
Yes, would be nice to add but I'm a bit strapped for time at the moment. Will try to have a look!

**Comment by zizther** [comment]
No immediate need from my end, I have been looking at alternatives to Algolia, TypeSense seems to be a suitable solution. MeiliSearch is also a popular alternative, but is a few features behind at the moment, but under development and are opening up a cloud hosted version alongside their self hosted version in 2022.

**Comment by acalvino4** [comment]
+1

**Comment by angusfretwell** [comment]
@janhenckens Have you done any thinking on this? I'm after Typesense support for a project so would be happy to help implement it.

**Comment by janhenckens** [comment]
Hey @angusfretwell, don't really have the bandwidth to pull this off at the moment but I'd be happy to review any PR's you can make for this.

**Comment by janhenckens** [comment]
I just came across this Typesense plugin that's also available on the store https://plugins.craftcms.com/typesense?craft4 - would that cover what you are looking for @angusfretwell?
